content/en/contribute/medic/product-core-competencies/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Product Team Core Competencies"
+linkTitle: "Product Team Core Competencies"
+weight: 5
+description: >
+  These core competencies are intended to guide Product Teammates in how everyone shows up for each other every day. They are to be used in hiring new teammates, teammate feedback, and regular manager check ins. 
+---
+
+## Reliable
+Gets things done when people expect them without prompting.
+
+## Team Player
+Acts in the best interest of the team and actively looks for ways to help the team. Makes time to support teammates to be successful.
+
+## Growth Mindset
+Always seeking to improve. Open minded, teachable, and coachable.
+
+## Proactive
+Sees things that need doing and takes action to keep things moving and make the team successful.
+
+## Effective Communicator
+Communicates regularly, openly, and effectively using the appropriate channels.
```

Comment:
We did have one unresolved request for a different word than "Communicates" but no suitable replacement yet. I'm going to move forward with what we have for now on this line, but I think everyone is open to changes going forward as we find better wording.